babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -50,4 +50,28 @@ SqlNode DateFunctionCall() :
 |   < TILDE: "~" >
 }
 
+SqlBinaryOperator SpecialBinaryOperator() :
+{
+}
+{
+    <PG_CAST> { return SqlStdOperatorTable.PG_CAST; }
+}
```

Comment:
You can replace `SqlStdOperatorTable.PG_CAST` with `SqlStdOperatorTable.CAST` cause they have the same semantics.